src/components/MessageSender.js

```diff
@@ -1,7 +1,73 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import styled from 'styled-components';
 
-export default function MessageSender() {
-	return <Wrapper>메시지 전송부</Wrapper>;
+export default function MessageSender(props) {
+	
+	const [message, setMessage] = useState('');
+	
+	const inputTagStyle = {	// 메시지 입력칸
+		width: '1150px',
+		height: '60%',
+		paddingLeft: '12px',
+		borderRadius: '20px 20px 20px 20px',
+		border: '1px solid rgba(0, 0, 0, 0.3)',
+		background: 'rgba(255, 255, 255, 0.3)',
+		outline: 'none',
+
+		display: 'inline-flex',		// inline으로해야 input과 button이 같은 선상에 있다.
+		margin: '10px'
+	};
+
+	const inputButtonTagStyle = {	// 전송 버튼
+		display: 'inline-flex',
+		height: '50px',
+		width: '65px',
+		borderRadius: '35%',
+		backgroundColor: '#FFE066',
+		border: 'none',
+		outline: 'none',
+
+		textAlign: 'center',
+		justifyContent: 'center',
+		fontSize: '15px'
+	}
+
+	// 입력받은 메시지를 ChattingScreen로 전달.
+	const formSubmit = e => {
+		e.preventDefault();
+		props.onSubmit(message);
+		document.querySelector('#messageBox').value = '';
+		setMessage('');		// 메시지 한번 전송 후, 메시지state를 초기화 해야함.
+	}
+
+	const onChangeMessage = e => {
```

Comment:
- 함수 이름을 지을 때는 영어의 기본적 문법과 문장 구조, 어순을 따라주세요! 
onChangeMessage => onMessageChange

- message가 아니라 messageInput이 바뀌는 거니까, 특정해주세요~
onMessageChange => onMessageInputChange

- 부사/전치사구가 아닌 아닌 동사형으로 지어주세용
handleMessageInputChange

```suggestion
	const handleMessageInputChange = e => {
```